apps/ledger-live-desktop/tests/specs/countervalues/api.spec.ts

```diff
@@ -0,0 +1,31 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+
+test.use({ userdata: "1AccountBTC1AccountETH" });
+
+test.use({
+  env: {
+    // disable mock for countervalues
+    MOCK_COUNTERVALUES: "",
```

Comment:
Why is it a boolean in the file above (common.ts) and an empty string here. Shouldn't it be false ?

Reply:
indeed it's weird that it was sometimes set a value to `true`, I basically did exactly like we do for `MOCK`, in MOCK case, the env is a string that is the "seed" of the mock generator. we could have same principle on MOCK_COUNTERVALUES even tho today it's only used as a on/off.
the `true` is going to actually be a `"true"` seed 😆 
anyway, nothing i really changed on the MOCK paradigm, as you can see it's parsed with `stringParser` like for the env MOCK

Reply:
As far as I know, environment variables are always strings so I would say it does not matter at all and true gets stringified to "true".

Reply:
live common env.ts still have some sort of comprehension of string->boolean, basically will check if it's "true" or "false" to match to booleans **if** an env is defined as boolean. which is not the case of MOCK_COUNTERVALUES atm

Reply:
ah good to know 👍

Reply:
Ok thanks :)